apps/vanilla-app/README.md

```diff
@@ -4,19 +4,19 @@
 
 You need to download assets from npm before running the examples:
 
-```bash
+```
```

Comment:
May I ask the reason removing `bash` here?

Reply:
Good question. the `bash` annotation is useful only in bash scripts, i.e. files that end in .sh etc.  This command I'm removing the bash `bash` annotation from is a global terminal command, i.e. it could be run on Windows, Mac or Linux shell. The annotation colors sometimes some commands that have no correlation to the global terminal commands.

Reply:
Got it, but I think it is still better to keep his for basic syntax highlight. I would suggest to keep it.

Reply:
Sure thing. If my reasoning was not good enough I'll change it back as you wish.

Reply:
Just make sure to squash the commits if you're the one merging!